movie/src/components/MainContainer/index.jsx

```diff
@@ -6,17 +6,29 @@ import movies from "../Json/Movies.json";
 
 const MainContainer = () => {
   const [movieList, setMovieList] = useState(movies);
-  const handleOnGetMoviesAfterDoAction = (moviesFromSorting) => {
+  // fourIncreasement, I use it for set default 4 to display 4 movies and increase 4 when click on add more button.
+  const [fourIncreasement, setFourIncreasement] = useState(4);
```

Comment:
1. Please find a better name for `fourIncreasement`
2. create a const for that `4` value. and use it with the `useState` and `onLoadMore`. 

P.S https://codehs.com/glossary/term/14. this will tell why you should have it as a const